src/status_im/contexts/wallet/common/sheets/remove_account/view.cljs

```diff
@@ -79,13 +86,17 @@
    [rn/view {:style style/desc-container}
     [quo/text {:weight :medium}
      (i18n/label :t/remove-watched-address-desc)]]
-   [footer {:submit-disabled? false}]])
+   [footer
+    {:submit-disabled? false
+     :address          address
+     :toast            (i18n/label :t/watched-account-removed)}]])
```

Comment:
consider :toast-label/ :toast message instead. 
Toasts are generally an object of data and I was more expecting to see that here

Reply:
Replaced with :toast-message